src/main/java/me/fril/modblocker/ModBlocker.java

```diff
@@ -48,5 +52,10 @@ public static void handleDisconnect(EntityPlayerMP entityPlayerMP, String modid)
         String disconnectMessage = MBConfig.config.disconnectMessage.replaceAll("@MOD", modid);
         entityPlayerMP.connection.disconnect(new TextComponentString(disconnectMessage));
     }
+
+    public static void handleBan(EntityPlayerMP entityPlayerMP, String modid){
+        UserListBansEntry userlistbansentry = new UserListBansEntry(entityPlayerMP.getGameProfile(), (Date)null, MODID, (Date)null, MBConfig.config.banMessage);
```

Comment:
Have you tested these values being null?

Reply:
Check CommandBanPlayer from Minecraft, it's almost the same xD

Reply:
But yes, I can fill these parameters correctly, let me handles that

Reply:
null is just here to put the ban permanent I think

Reply:
Alright, cool cool

Reply:
then we could push the thing and define a ban duration